preferences/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
```

Comment:
Looks like it's fixed in Gradle 8.1 🎉 

https://youtrack.jetbrains.com/issue/KTIJ-19369/False-positive-cant-be-called-in-this-context-by-implicit-receiver-with-plugins-in-Gradle-version-catalogs-as-a-TOML-file